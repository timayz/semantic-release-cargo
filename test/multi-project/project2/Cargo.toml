[package]
name = "project2"
version = "0.2.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
project1 = { path = "../project1", features = ["event"] }
lb1 = "1.0.2"

[dev-dependencies]
project1 = { path = "../project1", features = ["event", "test"] }
