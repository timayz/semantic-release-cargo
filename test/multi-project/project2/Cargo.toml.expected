[package]
name = "project2"
version = "1.33.7"
edition = "2021"

[dependencies]
project1 = { path = "../project1", features = ["event"], version = "1.33.7" }
lb1 = "1.0.2"
